jobs/ci-kubernetes-node-kubelet-dockerized.sh

```diff
@@ -22,8 +22,8 @@ set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
 
-export NODE_TEST_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh"
-export NODE_TEST_PROPERTIES="test/e2e_node/jenkins/jenkins-ci.properties"
+export NODE_TEST_SCRIPT="${REPO_DIR}/test/e2e_node/jenkins/e2e-node-jenkins.sh"
```

Comment:
https://github.com/kubernetes/test-infra/blob/master/jenkins/node-dockerized.sh#L49

it's mapped but you are right it's pointing to outside where I want. Just made it kubernetes.